scripts/test-templates.sh

```diff
@@ -51,6 +51,7 @@ dotnet new stacks-cqrs -n Cqrs.AllTheThings -do Menu --cloudProvider Azure --cic
 dotnet new stacks-cqrs -n Cqrs.ServiceBus -do Menu --cloudProvider Azure --cicdProvider AZDO -e ServiceBus
 dotnet new stacks-cqrs -n Cqrs.Dynamo -do Menu --cloudProvider AWS -db DynamoDb  
 dotnet new stacks-cqrs -n Cqrs.Sns -do Menu --cloudProvider AWS -e AwsSns
+dotnet new stacks-cqrs -n Cqrs.EventHub -do Menu --cloudProvider GCP --cicdProvider GHA -db DynamoDb -e EventHub
```

Comment:
We do not currently support GCP, is this line needed?

Reply:
That is purely to generate the DevOps pipelines. The key purpose of this new template is EventHub. That was untested and therefore needed to be added.